npm/react/plugins/babel/index.d.ts

```diff
@@ -0,0 +1,27 @@
+import { Configuration } from "webpack";
+
+interface AdditionalOptions {
+	/**
```

Comment:
Is this indentation correct?

Reply:
impossible to know without prettier...
I'll replace with 2 spaces